todo_app/data/item.py

```diff
@@ -0,0 +1,9 @@
+class Item:
+    def __init__(self, id, task, status = 'To Do'):
+        self.id = id
+        self.task = task
+        self.status = status
+
+    @classmethod
+    def from_trello_cards(cls, list):
+        return cls(list['id'], list['name'])
```

Comment:
I think that's a card not a list?
```suggestion
    @classmethod
    def from_trello_cards(cls, card):
        return cls(card['id'], card['name'])
```